FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo/Images.xcassets/LaunchImage.launchimage/Contents.json

```diff
@@ -4,15 +4,134 @@
       "orientation" : "portrait",
       "idiom" : "iphone",
       "extent" : "full-screen",
+      "minimum-system-version" : "8.0",
```

Comment:
Yeah, one of those things that changed just by touching it. Didn't think it
was worth messing with git to revert it.

On Fri, Jan 8, 2016 at 12:51 Patrick Chamelo <notifications@github.com>
wrote:

> In
> FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemo/Images.xcassets/LaunchImage.launchimage/Contents.json
> <https://github.com/facebook/ios-snapshot-test-case/pull/137#discussion_r49236613>
> :
>
> > @@ -4,15 +4,134 @@
> >        "orientation" : "portrait",
> >        "idiom" : "iphone",
> >        "extent" : "full-screen",
> > +      "minimum-system-version" : "8.0",
>
> I guess this file was introduced for the device size right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ios-snapshot-test-case/pull/137/files#r49236613>
> .
>